torch/_higher_order_ops/flex_attention.py

```diff
@@ -179,11 +179,12 @@ def trace_flex_attention(
     ] + [torch.zeros((), dtype=torch.int) for _ in range(4)]
     with TransformGetItemToIndex():
         score_graph = make_fx(score_mod)(*example_vals, *other_buffers)
-    proxy_mode.tracer.root.register_module("sdpa_score", score_graph)
+    qualname = proxy_mode.tracer.get_fresh_qualname("sdpa_score")
```

Comment:
would we hard error before or just overrwrite?

Reply:
silently incorrect.